src/components/CreatePost/CreatePost.tsx

```diff
@@ -49,64 +54,222 @@ const CreatePost = () => {
     }));
   };
 
-  const firestore = useContext(FirestoreContext);
+  const { firestore, storage } = useContext(FirebaseContext);
+
+  const [title, setTitle] = useState<string>('');
+  const [description, setDescription] = useState<string>('');
+  const [selectedFiles, setSelectedFiles] = useState<FilesInterface[]>([]);
+  const [selectedImages, setSelectedImages] = useState<ImagesInterface[]>([]);
```

Comment:
Not sure why we have SelectedFiles as well as SelectedImages since they hold similar data.